pkgs/by-name/co/collabora-online/package.nix

```diff
@@ -0,0 +1,121 @@
+{
+  autoreconfHook,
+  cairo,
+  cppunit,
+  fetchFromGitHub,
+  fetchNpmDeps,
+  lib,
+  libcap,
+  libpng,
+  libreoffice-collabora,
+  nodejs,
+  npmHooks,
+  pam,
+  pango,
+  pixman,
+  pkg-config,
+  poco,
+  python3,
+  rsync,
+  stdenv,
+  zstd,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "collabora-online";
+  version = "24.04.5-3";
+
+  src = fetchFromGitHub {
+    owner = "CollaboraOnline";
+    repo = "online";
+    rev = "refs/tags/cp-${finalAttrs.version}";
+    hash = "sha256-wy+vfRmKYULq2kg0MKhdxTD7HC63cTd1cYX5xN7LYgg=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    libcap
+    nodejs
+    npmHooks.npmConfigHook
+    pkg-config
+    python3
+    python3.pkgs.lxml
+    python3.pkgs.polib
+    rsync
```

Comment:
libcap goes to buildInputs

Reply:
Removed as it is not required after a bump to `24.04.6-1`.